sqrl-calcite/src/main/java/com/datasqrl/calcite/QueryPlanner.java

```diff
@@ -203,9 +203,15 @@ public RelDataType parseDatatype(String datatype) {
       return this.cluster.getTypeFactory().createSqlType(SqlTypeName.TIMESTAMP, 3);
     }
 
+    if (datatype.equalsIgnoreCase("BigInteger")) {
+      return this.cluster.getTypeFactory().createSqlType(SqlTypeName.BIGINT);
```

Comment:
We could use the graphql-java base set of scalars (`scalar GraphQLBigInteger`) so all of the types can be mapped uniformly. https://github.com/graphql-java/graphql-java-extended-scalars?tab=readme-ov-file#java-primitives

Reply:
What do you mean ?
`scalar GraphQLBigInteger` maps to `"BigInteger"`  String litteral in [JavaPrimitives](https://github.com/graphql-java/graphql-java-extended-scalars/blob/9e283115657220fd191e7a03d9ecf07964746aed/src/main/java/graphql/scalars/java/JavaPrimitives.java#L373)

Reply:
I pushed comments and test related to that

Reply:
Calling it BigInteger may be confusing since there is also a "BigInt" that is a java Long type. The Numeric type is the type that maps to java BigInteger. Since we have one type system for both sql and graphql, it may be better to disambiguate this by calling it "GraphQLBigInteger"

Reply:
Thanks it is clearer. I agree. I pushed a fix. PTAL